src/yamlfix/model.py

```diff
@@ -0,0 +1,23 @@
+"""Define program entities like configuration value entities."""
+
+from typing import Optional
+
+from maison.schema import ConfigSchema
+
+
+class YamlfixConfig(ConfigSchema):
+    """Configuration entity for yamlfix."""
+
+    allow_duplicate_keys: bool = True
+    config_path: Optional[str] = None
+    explicit_start: bool = True
+    flow_style_sequence_multiline: bool = False
+    flow_style_sequence: Optional[bool] = None
```

Comment:
I'd say we use `flow_style_sequence: bool = True` as default, what do you think?